scripts/simplify.py

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/python3
+
+import sys
+import getopt
+import uuid
+from xml.dom import minidom
+
+default_layer_attrs = {
+	'inkscape:groupmode': 'layer',
+	'sodipodi': 'true', # Force layer locked by default
+	'style': 'display:inline'
+}
+
+default_trace_style = 'fill:none;stroke:#ff0000;stroke-width:1.25;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1'
+default_zone_style = 'opacity:0.5;fill:#00ffff;stroke:#000000;stroke-width:0.99999994px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1'
+default_zone_label = 'Copper Fill'
+
+def sort_element_attr(element):
+	attributes = element.attributes.items()
+
+	# Remove attributes from element
+	keys = list(element.attributes.keys()) if element.attributes else []
+	for attribute in keys:
+		element.removeAttribute(attribute)
+
+	# Sort
+	attributes.sort(key=lambda name: name[0])
+
+	# Add attributes back
+	for name, value in attributes:
+		element.setAttribute(name, value)
+
+def process_image_layer(xml, layer, new_layer):
+	images = layer.getElementsByTagName('image')
+
+	image_allowed_attrs = ['xlink:href', 'y', 'x', 'preserveAspectRatio', 'height', 'width']
+	for image in images:
+		new_image = xml.createElement('image')
+
+		# Recreate each allowed attribute in the newly created element
+		for allowed_attr in image_allowed_attrs:
```

Comment:
In addition to warning, we probably want to explicitly error on certain attributes that we recognize as problematic:

- allowed attribs: will be kept, anything else is at least a warning.
- disallowed attribs: will be error.

*(Could be extended with something where we set allowed defaults, so we'd tolerate some tags if they have a no-op argument; thereby avoiding errors or warnings)*